docs/Changelog.md

```diff
@@ -18,6 +18,7 @@ Nussknacker versions
 * [#949](https://github.com/TouK/nussknacker/pull/949) JVM options can be configured via JDK_JAVA_OPTIONS env variable (in docker and standalone distribution) 
 * [#954](https://github.com/TouK/nussknacker/pull/954) Correct handling of types in empty inline lists 
 * [#944](https://github.com/TouK/nussknacker/pull/903) Pluggable SchemaRegistry Cache 
+* [#958](https://github.com/TouK/nussknacker/pull/958) Speedup getting avro schema
```

Comment:
It is not exactly "speedup", because without caching it was already fast :) Also it is very minor change, let's keep this list concise

Reply:
Hmm, I think we should add all changes in changelog. Moreover I've added there some improvements to cache. So I can agree with you that this is not speed up, but still it's some good improvement ;) I will change description to: "Improvements Avro Cache"